Libraries/src/Amazon.Lambda.Logging.AspNetCore/LambdaLoggerOptions.cs

```diff
@@ -221,9 +222,18 @@ private static bool TryGetSection(IConfiguration configuration, string key, out
                 }
             }
 
-            // Extract a reverse sorted list of wildcard categories.  This allows us to quickly search for most specific match
+            // Extract reverse sorted list of given categories. This will allow us to quickly search for most specific namespace prefix match
+            // to least specific - notice we add "." (namespace delimiter) at the end, assuming it is namespace prefix.
+            var namespacePrefixCategories = logLevelsMapping.Keys

```

Comment:
Seems like with this change we are treating every namespace as a wild card if there is not exact match. Which makes sense to me but instead of building 2 separate collection which lives the awkward question which collection takes priority should we just build one sorted collection?

Reply:
@normj good point, I was too afraid to touch existing code, I will prepare change.